test/jdk/javax/swing/SwingUtilities/bug4967768.java

```diff
@@ -37,13 +37,19 @@
 
 public class bug4967768 {
     private static final String INSTRUCTIONS = """
-            When the test starts you'll see a button "Oops"
-            with the "p" letter underlined at the bottom
-            of the instruction frame.
+            When the test starts you'll see a button "Oops".
+
+            For Windows and GTK Look and Feel, you will need to
+            press the ALT key to make the mnemonic visible.
+            Once the ALT key is pressed, the letter "p" will be
+            underlined at the bottom of the instruction frame.
 
             Ensure the underline cuts through the descender
```

Comment:
Maybe you can get rid of the part mentioning descender and just say "The letter p cuts through the underline".